tests/DwitTech.AccountService.WebApi.Tests/Controllers/UserControllerTest.cs

```diff
@@ -353,5 +353,81 @@ public async Task Logout_Should_Return_False_If_AuthorizationHeader_Is_Not_Given
             Assert.IsType<BadRequestObjectResult>(result);
 
         }
+
+        [Fact]
+        public async Task EditUser_Should_Return_True_If_Successful()
+        {
+            var userService = new Mock<IUserService>();
+            var authService = new Mock<IAuthenticationService>();
+            var actService = new Mock<IActivationService>();
+            var iloggerMock = new Mock<ILogger<UserController>>();
+            string authHeader = "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJVc2VySWQiOiIxIiwiaHR0cDovL3NjaGVtYXMueG1sc29hcC5vcmcvd3MvMjAwNS8wNS9pZGVudGl0eS9jbGFpbXMvZW1haWxhZGRyZXNzIjoidXNlckBleGFtcGxlLmNvbSIsImh0dHA6Ly9zY2hlbWFzLnhtbHNvYXAub3JnL3dzLzIwMDUvMDUvaWRlbnRpdHkvY2xhaW1zL2dpdmVubmFtZSI6IkphbWVzIiwiaHR0cDovL3NjaGVtYXMueG1sc29hcC5vcmcvd3MvMjAwNS8wNS9pZGVudGl0eS9jbGFpbXMvc3VybmFtZSI6IkpvaG4iLCJodHRwOi8vc2NoZW1hcy5taWNyb3NvZnQuY29tL3dzLzIwMDgvMDYvaWRlbnRpdHkvY2xhaW1zL3JvbGUiOiJVc2VyIiwiZXhwIjoxNjgxNDk3NTA2LCJpc3MiOiJ0ZXN0SXNzdWVyIn0.uaiS1np_dt7-DPr2ot5JLf_fffdeT0iza83al8n7jNM";
+            var editRequestDto = new EditRequestDto
+            {
+                FirstName = "john",
+                LastName = "doe",
+                AddressLine1 = "south east london",
+                AddressLine2 = "north carolina",
+                PhoneNumber = "09085678900",
+                PostalCode = "90021",
+                ZipCode = "20017",
+                City = "reo",
+                Country = "united nations",
+                Email = "example@gmail.com",
+                State = "washington dc"
+            };
+            userService.Setup(x => x.EditAccount(authHeader, editRequestDto)).Returns(Task.FromResult(true));
+            var httpContext = new DefaultHttpContext();
+            var request = httpContext.Request;
+            request.Method = "PUT";
+            request.Headers["Authorization"] = authHeader;
+            var controller = new UserController(actService.Object, authService.Object, userService.Object, iloggerMock.Object);
+            controller.ControllerContext = new ControllerContext()
+            {
+                HttpContext = httpContext
+            };
+            var result = await controller.EditAccount(editRequestDto);
+            Assert.IsType<OkObjectResult>(result);
```

Comment:
Your test name hints at the fact that this test should assert that the method returns true, but there's no assert statement that covers that.

Reply:
> Your test name hints at the fact that this test should assert that the method returns true, but there's no assert statement that covers that.

Your test name should return Ok since your assert is returning ok